sql.txt

```diff
@@ -1,14 +1,39 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
 
+SELECT CustomerName, ContactName, Country
+FROM Customers
+ORDER BY Country
+LIMIT 20
+OFFSET 10;
+
```

Comment:
This isn't quite right. For limit it needs to be 10, it  will limit your set to 10 rows, and it will be offset by the first 10, leaving 11 - 20.